server/src/com/cloud/storage/StorageManagerImpl.java

```diff
@@ -1719,6 +1719,7 @@ public boolean storagePoolHasEnoughSpace(List<Volume> volumes, StoragePool pool,
         }
 
         // allocated space includes templates
+        s_logger.debug("Destination pool id: " + pool.getId());
```

Comment:
Please wrap this `DEBUG` log in an `if (s_logger.isDebugEnabled)` check to prevent unnecessary/expensive string concatenation when `DEBUG` logging is not enabled.  Also, please add context to message to explain what operation is being performed for the destination pool.